artemis/modules/crtsh.py

```diff
@@ -53,22 +59,33 @@ def query_sql(self, domain: str) -> set[str]:
         return ct_domains
 
     def query_json(self, domain: str) -> set[str]:
-        ct_domains: set[str] = set()
         response = requests.get(f"https://crt.sh/?q={domain}&output=json")
         if response.ok:
+            ct_domains: set[str] = set()
             for cert in response.json():
                 for entry in cert["name_value"].split("\n"):
                     if re.fullmatch(DOMAIN_REGEX, entry):
                         ct_domains.add(entry)
-        return ct_domains
+            return ct_domains
+        else:
+            raise UnableToObtainSubdomainsException()
```

Comment:
Seems like could just use `response.raise_for_status()` instead of inventing our own exceptions :thinking: